### PR TITLE
New category for changeset viewers/review tools

### PR DESCRIPTION
**Pull request by Woazboat**
Move changeset viewers/review tools into a new category, separate from the general tools section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -97,8 +97,19 @@ This list contains projects using OpenStreetMap data for creative purposes, as w
 
 * [Baremaps](https://www.baremaps.com/) - Open source pipeline for producing Mapbox vector tiles from OpenStreetMap with Postgis and Java. ([Source Code](https://github.com/baremaps/baremaps))
 
+## Changeset Review & Viewer Tools
```
Comment (Unkn0wnCat) [inline README.md:100]: I think this should just be "Changeset Tools" so the title will readable at a glance

```suggestion
## Changeset Tools
```

File: README.md [hunk 2]
```diff
@@ -18,6 +18,9 @@ This list contains projects using OpenStreetMap data for creative purposes, as w
   * [Web Tools](#web-tools)
   * [Mobile Tools](#mobile-tools)
   * [Native Tools](#native-tools)
+* [Changeset Review & Viewer Tools](#changeset-review--viewer-tools)
```
Comment (Unkn0wnCat) [inline README.md:21]: See below.
```suggestion
* [Changeset Tools](#changeset-tools)
```

File: README.md [hunk 3]
```diff
@@ -97,8 +97,19 @@ This list contains projects using OpenStreetMap data for creative purposes, as w
...
+## Changeset Review & Viewer Tools
+
+### Web Services
+
+* [osmcha](https://osmcha.mapbox.com) - Detector for suspicious changesets. ([Source Code](https://github.com/mapbox/osmcha-frontend))
+* [achavi](https://overpass-api.de/achavi/) - Augmented change viewer. ([Source Code](https://github.com/nrenner/achavi/))
+
 <!--Offline, may be because of russia-ukraine-war - * [whodidit](http://zverik.osm.rambler.ru/whodidit/) - Changesets analyzer.-->
 
+### Selfhosted
```
Comment (Unkn0wnCat) [inline README.md:109]: In my opinion this should be a part of "Web Services", as almost all of the web services can be self-hosted and adding a separate category will just complicate things

**Review by Unkn0wnCat — CHANGES_REQUESTED** [review]
Great idea! That category will definitely help keep things more organized. I do however have a few comments as attached in this review!

File: README.md (hunk 3 above)
Comment (Woazboat) [reply README.md:109]: The difference is that the 'web service' ones provide a hosted instance and can be used by anyone without any extra work while the 'selfhosted' ones do not and need to be set up on your own to be usable (which is a major hurdle that excludes 95% of users)

Comment (Unkn0wnCat) [reply README.md:109]: I mean for the project you've listed under Selfhosted there would be a hosted instance, but it seems to be defunct. (expandable.dk seems to not have any A, AAAA or MX DNS records)

I still think a "Selfhosted" category would be irritating, as that would imply the others would not be self-hostable, but I agree osm-analytic-tracker does not really fit with the other web-services.

Maybe we should just leave osm-analytic-tracker out for now until there is a working instance again?

Comment (Unkn0wnCat) [reply README.md:109]: Or we could add a note like with a footnote explaining the situation as follows:

(⚠️ Needs to be self-hosted[^no-running-instances])

[^no-running-instances]: Some projects have no running instances and need to be deployed to a custom server to be used. Please refer to the projects website to learn more about setting up an instance.

Comment (Woazboat) [reply README.md:109]: Adding some sort of note would be a good idea. Not adding self-host-only tools kind of defeats the purpose of this list.

Comment (Unkn0wnCat) [reply README.md:109]: I agree not adding those tools would not be helpful.

I think the note with a footnote explaining further is the best option, so if you'd add that I'll merge this! 😃

Comment (Woazboat) [reply README.md]: I added the footnote and a bunch more tools

File: README.md [hunk 4]
```diff
@@ -100,7 +99,24 @@ We also have a list of <a href="UNMAINTAINED.md">unmaintained projects</a>. If y
...
+* [changeset-map](http://osmlab.github.io/changeset-map/) - Map visualizing changes in selected changesets. Used in OsmCha. ([Source Code](https://github.com/osmlab/changeset-map))
+* [OSM History Viewer](https://osmhv.openstreetmap.de/) - Changeset visualization. ([Source Code](https://github.com/osmrmhv/osmrmhv))
+* [Who did it?](https://simon04.dev.openstreetmap.org/whodidit/) - Map highlighting recent edits per tile. Provides RSS feeds for watching changes in custom bounding boxes.  ([Source Code](https://github.com/simon04/whodidit) / [Wiki](https://wiki.openstreetmap.org/wiki/Quality_assurance#WhoDidIt))
 <!--Offline, may be because of russia-ukraine-war - * [whodidit](http://zverik.osm.rambler.ru/whodidit/) - Changesets analyzer.-->
+* [Osm Change Tiles](https://resultmaps.neis-one.org/osm-change-tiles) - Map highlighting recent edits per tile. Provides RSS feeds for watching changes in custom bounding boxes.
+* [Analytic OSM Tracker](https://github.com/MichaelVL/osm-analytic-tracker)[^selfhosted-only] - Tracker for changesets in your region. ([Source Code](https://github.com/MichaelVL/osm-analytic-tracker))
```
Comment (Unkn0wnCat) [inline README.md:115]: This currently produces a double-link error

```suggestion
* [Analytic OSM Tracker](https://github.com/MichaelVL/osm-analytic-tracker)[^selfhosted-only] - Tracker for changesets in your region.
```

**Review by Unkn0wnCat — CHANGES_REQUESTED** [review]
Looks pretty good, however the linter does not like the link being double, and you need to add `<!-- lint disable no-undefined-references -->` towards the top (preferably a line after the ToC) as the linter does not support footnotes by default.

File: README.md (hunk 4 above)
Comment (Woazboat) [reply README.md]: I added the double link for consistency so it doesn't seem like the project doesn't provide any source code, but I removed it now (and also added the undefined references linter annotation)

**Review by Unkn0wnCat — APPROVED** [review]
Looking good

**Comment by Unkn0wnCat** [comment]
Thanks for doing this! I know it was a lot of work and I requested a lot of changes! 🎉

**Pull request merged**